### PR TITLE
feat: dashboard project size

### PR DESCRIPTION
**Pull request by datphamcode295**
#### What's this PR do?

- [x] project sizes for dashboard

#### What are the relevant Git tickets?
https://www.notion.so/GET-API-to-fetch-project-size-to-render-pipe-chart-dc2bcaf2d6cf491abcdb8f1ed49dcfe7

#### Screenshots (if appropriate)
image

#### Request
```
curl --location --request GET 'http://localhost:8200/api/v1/dashboards/project-sizes' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2OTkzMjExNDIsImlkIjoiMjY1NTgzMmUtZjAwOS00YjczLWE1MzUtNjRjM2EyMmU1NThmIiwiYXZhdGFyIjoiaHR0cHM6Ly9zMy1hcC1zb3V0aGVhc3QtMS5hbWF6b25hd3MuY29tL2ZvcnRyZXNzLWltYWdlcy81MTUzNTc0Njk1NjYzOTU1OTQ0LnBuZyIsImVtYWlsIjoidGhhbmhAZC5mb3VuZGF0aW9uIiwicGVybWlzc2lvbnMiOlsiZW1wbG95ZWVzLnJlYWQiXSwidXNlcl9pbmZvIjpudWxsfQ.GENGPEucSUrILN6tHDKxLMtj0M0REVMUPC7-XhDMpGM' \
--data-raw ''
```

#### Response
```
{
    "data": [
        {
            "id": "8dc3be2e-19a4-4942-8a79-56db391a0b15",
            "name": "Fortress",
            "code": "fortress",
            "size": 5
        }
    ]
}
```

### CODE REVIEW

File: migrations/seed/permissions.sql [hunk 1]
```diff
@@ -39,4 +39,8 @@ INSERT INTO public.permissions (id, deleted_at, created_at, updated_at, name, co
 ('f632f18c-98da-4c1a-b2c6-9ccd0c7c5a39', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Employee Roles Create', 'employeeRoles.create'),
 ('5c5919b5-e74b-4854-bdf6-b91396a8317c', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Employee Roles Read', 'employeeRoles.read'),
 ('68fe5bcf-38ae-446a-8cbb-8f88b5c6eb44', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Employee Roles Edit', 'employeeRoles.edit'),
-('98744d43-f14b-4235-8dfc-5265668daa21', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Employee Roles Delete', 'employeeRoles.delete');
+('98744d43-f14b-4235-8dfc-5265668daa21', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Employee Roles Delete', 'employeeRoles.delete'),
+('4be48741-f8b2-4562-964e-65c081fec7a9', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'DashBoard Create', 'dashboards.create'),
```
Comment (nguyennh4522) [inline migrations/seed/permissions.sql:43]:
```suggestion
('4be48741-f8b2-4562-964e-65c081fec7a9', NULL, '2022-11-11 18:34:14.743263', '2022-11-11 18:34:14.743263', 'Dashboard Create', 'dashboards.create'),
```

File: pkg/handler/dashboard/dashboard.go [hunk 2]
```diff
@@ -25,3 +29,29 @@ func New(store *store.Store, repo store.DBRepo, service *service.Service, logger
...
+// ProjectSizes godoc
+// @Summary Get the total number of active member in each project
+// @Description Get the total number of active member in each project
+// @Tags Dashboard
+// @Accept json
+// @Produce json
+// @Param Authorization header string true "jwt token"
+// @Success 200 {object} view.ProjectSizeResponse
+// @Failure 500 {object} view.ErrorResponse
+// @Router /dashboards/project-sizes [get]
```
Comment (nguyennh4522) [inline pkg/handler/dashboard/dashboard.go:42]:
```suggestion
// @Router /dashboards/projects/sizes [get]
```
I think we should separate routes for each part in dashboard

File: pkg/store/dashboard/dashboard.go [hunk 3]
```diff
@@ -1,7 +1,24 @@
...
 type store struct{}
 
 func New() IStore {
 	return &store{}
 }
+
+func (s *store) GetProjectSizes(db *gorm.DB) ([]*model.ProjectSize, error) {
+	var ru []*model.ProjectSize
+
+	query := `
```
Comment (nguyennh4522) [inline pkg/store/dashboard/dashboard.go:17]: format it

Comment (datphamcode295) [reply pkg/store/dashboard/dashboard.go:17]: updated

**Review by thanhpn — APPROVED** [review]

**Pull request merged**